spread.yaml

```diff
@@ -758,6 +758,7 @@ restore-each: |
 suites:
     tests/lib/tools/suite/:
         summary: Tests for tests/lib/tools tools
+        backends: [google]
```

Comment:
Would it make sense to allow this also for the qemu backend?

Reply:
yes, it should not be a problem. At some point if we need to run all the tests in s390x, we will need to skip on that architecture.